spec/shinq/client_spec.rb

```diff
@@ -34,13 +34,13 @@
 
   describe '.column_names' do
     it 'fetches column_names' do
-      expect(shinq_client.column_names(table_name: :queue_test)).to eq(['job_id', 'title', 'scheduled_at', 'enqueued_at'])
+      expect(shinq_client.column_names(table_name: :queue_test).sort).to eq(['job_id', 'title', 'scheduled_at', 'enqueued_at'].sort)
```

Comment:
These changes are made to pass tests with MySQL 8.0 in my local environment.

FYI: These tests fail with MySQL 8.0 even before this PR is made.